docs/src/Combinatorics/graphs.md

```diff
@@ -56,10 +56,10 @@ complete_bipartite_graph(n::Int64, m::Int64)
 edges(g::Graph{T}) where {T <: Union{Directed, Undirected}}
 has_edge(g::Graph{T}, source::Int64, target::Int64) where {T <: Union{Directed, Undirected}}
 has_vertex(g::Graph{T}, v::Int64) where {T <: Union{Directed, Undirected}}
+number_of_edges(g::Graph{T}) where {T <: Union{Directed, Undirected}}
+nv(g::Graph{T}) where {T <: Union{Directed, Undirected}}
```

Comment:
Why is `ne` renamed to `number_of_edges` but `nv` (and elsewhere in this file: `nvertices`) left as it is, without a `number_of_vertices` being added?

Reply:
This is an (more or less) alphabetic list, `nv` just comes at the bottom